daemon/cmld.c

```diff
@@ -1607,7 +1617,7 @@ cmld_container_stop(container_t *container)
 
 	DEBUG("Trying to stop container %s", container_get_description(container));
 
-	if (container_get_state(container) == COMPARTMENT_STATE_STOPPED) {
+	if (!container_is_stoppable(container)) {
```

Comment:
check in compartment_stop is sufficient -> remove here

Reply:
done